com.chartboost.mediation/Runtime/AdFormats/Banner/Unity/ChartboostMediationUnityBannerAd.cs

```diff
@@ -16,6 +17,14 @@ namespace Chartboost.AdFormats.Banner.Unity
     public delegate void ChartboostMediationUnityBannerAdEvent();
 
     public delegate void ChartboostMediationUnityBannerAdDragEvent(float x, float y);
+
+    public enum ResizeOption
+    {
+        FitHorizontal,
+        FitVertical,
+        FitBoth,
+        NoResize
```

Comment:
🤔 should we go with `StaticSize` or `AvoidResize`?? Just thinking if there's a more appropriate word instead of `NoResize`.

Reply:
`AvoidResize` sounds good 😄  or maybe `Disabled` 🤔  since the enum name is `ResizeOptions` so `ResizeOptions.Disabled`sound better ?

Reply:
Yeah, `Disable` may be the way to go.